### PR TITLE
Adds link to new docs to yoast/README.md

### PR DESCRIPTION
**Pull request by mburridge**
**What**:

Adds link to newly created docs page to yoast/README.md

**Why**:

The link was missing.

**How**:

**Tasks**:

- [x] Readme

**Unrelated Tasks**

- [ ] Code
- [ ] TSDocs
- [ ] TypeScript
- [ ] Unit tests
- [ ] End to end tests
- [ ] TypeScript tests
- [ ] Update starter themes
- [ ] Update other packages
- [ ] Update community discussions
- [ ] Add a changeset (with link to its [Feature Discussion](https://community.frontity.org/c/33) if it exists)

**Additional Comments**

### CODE REVIEW

**Comment by mburridge** [comment]
Why have all the google-ad-manager commits also shown up here????!!!!

**Review by DAreRodz — APPROVED** [review]
For me it's perfect, @mburridge.

PS: the link doesn't work, but I guess it's because it hasn't been published yet, right?

**Comment by mburridge** [comment]
@DAreRodz 

"PS: the link doesn't work, but I guess it's because it hasn't been published yet, right?" - yup, that's it. Thanks for your review of the [docs page](https://github.com/frontity/api-reference/pull/15), once @juanmaguitar has reviewed and approved the docs page then it will be published and the link will work. 😅

**Review by juanmaguitar — APPROVED** [review]

File: packages/yoast/README.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ export default {
 };
 ```
 
+> Comprehensive information about how to use the @frontity/yoast package can be found in the [docs](https://api.frontity.org/frontity-packages/features-packages/yoast.md).
```
Comment (cristianbote) [inline packages/yoast/README.md]: Should the link not have the `.md` extension? 🤔 
```suggestion
> Comprehensive information about how to use the @frontity/yoast package can be found in the [docs](https://api.frontity.org/frontity-packages/features-packages/yoast).
```

Comment (mburridge) [reply packages/yoast/README.md]: You're right - I've fixed it.

**Pull request merged**